tests/l2/dgpu/README.md

```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
   Attached to: #89 (PCI bridge)
 ```                        
 
+### Verify Media support for Intel® Data Center GPU
+It is only supported for Intel® Data Center GPU Flex Series.
+
+NOTE: Please note that the Intel media driver (https://github.com/intel/media-driver) library is compiled while building the image and contains implementations of various Media codecs. Those codecs are patented and are owned by their respective developers.
+
```

Comment:
this note is not so clear to me. Could you show me the original doc for the related information? Thanks!

Reply:
Not sure i understand the list of supported codecs are in the main readme of the project above. but if its confusing we can remove this note

Reply:
We should add the notes, Just need to make it very concise and clear. :-)

Reply:
removed that line as its confusing.